pdns/recursordist/docs/http-api/endpoint-cache.rst

```diff
@@ -12,6 +12,10 @@ Cache manipulation endpoint
 
   :query subtree: If set to `true`, also flush the whole subtree (default = `false`)
 
+  .. versionadded:: 4.4.0
+
+  :query type: If set only flush records of the specified type name.
```

Comment:
Needs a hint that this works only in the recursor, not auth

Reply:
It's rec specific doc, but fair enough.